migrations/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+
+from api_blog.app import get_config
+from api_blog.models import Base
+
+ALEMBIC_URL = get_config()["alembic_url"]
```

Comment:
it names `db_url`, isn't it?

Reply:
Those two urls have little different, so, I make two vars:
db_url: postgresql+asyncpg://kyrylo:password@localhost:5432/aiohttp_blog
alembic_url: postgresql://kyrylo:password@localhost/aiohttp_blog
First for sqlalchemy, second for alembic